WordPress/gutenberg
# Fixed Global Styles variables for colors, font family, gradient, fontSize

## What?

Fixes the font family, color, and gradient variables for Global Styles. Instead of saving the raw values, these values will now be persisted with the infix vars.
## Why?

fixes https://github.com/WordPress/gutenberg/issues/43397, https://github.com/Automattic/wp-calypso/issues/71641
## How?

We just need to update`STYLE_PATH_TO_CSS_VAR_INFIX` hardcoded list with the proper paths. 
## Testing Instructions





### Testing Instructions for Keyboard


- Checkout this branch on your local development environment
- Open the site editor
- Try to modify the font families, the colors (the theme predefined colors), gradients
- Repeat this for heading, links and buttons
- Make sure that the API request it makes on save is persisting the theme variables instead of the raw values
